
int32 index
int32 addr
int32 count
string[] valtype
---
int32 res
int32 index